src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -265,6 +265,7 @@ uct_cuda_ipc_md_open(uct_component_t *component, const char *md_name,
         .mkey_pack          = uct_cuda_ipc_mkey_pack,
         .mem_reg            = uct_cuda_ipc_mem_reg,
         .mem_dereg          = uct_cuda_ipc_mem_dereg,
+        .mem_query          = ucs_empty_function_return_unsupported,
```

Comment:
`(uct_md_mem_query_func_t)ucs_empty_function_return_unsupported`
and for other places where `ucs_empty_function_return_unsupported` used

Reply:
wasn't fixed? other fixes look good 👍

Reply:
Sorry missed this. Addressed this now.